String/union.py

```diff
@@ -0,0 +1,18 @@
+def union(list1,list2):
+    le=len(list1)
+    list1[le:le]=list2
+    list4=[]
+    list3=list(dict.fromkeys(list1))
+    for i in range(len(list3)):
+    	list4.append(int(list3[i]))
+    list4.sort()
+    for i in range(len(list4)):
+    	print(list4[i],end=" ")
+    
```

Comment:
This entire code can be made better, replace these list1, list2, list3, etc. with proper names so that anyone can understand it when they check out your code.